documentation/reference/github_contribution_practices.md

```diff
@@ -0,0 +1,75 @@
+# GitHub contribution practices
+
+Here are some of the practices we follow at Openverse when it comes to GitHub
+contributions.
+
+## GitHub labels
+
+### Aspect
+
+There are many [aspects](https://github.com/WordPress/openverse/labels?q=aspect)
+in which the Openverse project can be improved. These can directly involve the
+code but there are other related areas such as documentation and interfaces.
+
+### Technology
+
+Openverse projects span a lot of languages, frameworks, tools and
+[technologies](https://github.com/WordPress/openverse/labels?q=tech) across the
+stack from JavaScript and TypeScript on the frontend to Python on the backend.
+
+### Stack
+
+Openverse spans a tall
+[stack](https://github.com/WordPress/openverse/labels?q=stack), from the catalog
+at the innermost level, followed by the ingestion server, the API and the
+frontend. There is scope of improvement at each level.
+
+### Contributor friendliness
+
+We use two labels to identify issues that have been specifically created or
+selected for new contributors:
+["help wanted"](https://github.com/WordPress/openverse/issues?q=is:issue+is:open+sort:updated-desc+label:%22help+wanted%22)
+and "good first
+issue"](https://github.com/WordPress/openverse/issues?q=is:issue+is:open+sort:updated-desc+label:%22good+first+issue%22).
```

Comment:
This looks like the rendering got messed up!

```suggestion
and ["good first issue"](https://github.com/WordPress/openverse/issues?q=is:issue+is:open+sort:updated-desc+label:%22good+first+issue%22).
```

Reply:
Thanks for catching this, it was missing an opening `[`. Fixed in 7eef019.